docs/fendermint/observability.md

```diff
@@ -357,9 +357,9 @@ directory = "/path/to/log/directory"
 max_log_files = 5 # Number of files to keep after rotation
 rotation = "daily" # Options: minutely, hourly, daily, never
 ## Optional: filter events by domain
-domain_filter = "Bottomup, Consenesus, Mpool, Execution, Topdown, TracingError"
```

Comment:
looking for instances of `impl_traceables!`, I didn't find one with `"TracingError"`, but there was one with `"System"`